### ISSUE TITLE
Users no premium can't use /register

### ISSUE THREAD

**Issue by JavituRec** [kyngs/LibreLogin#228]
Hello,

Non-premium users can only use /reg on NanoLimbo server. When they try to use /register they get no permission message.

All the best.

**Comment by kyngs** [comment]
Not caused by librelogin as the /register command is not under permission. Perhaps there's another plugin interfering.

**Comment by JavituRec** [comment]
Yes, it is true. I just checked another plugin that blocked that command.

Thanks.
